pkg/utils/utils.go

```
@@ -78,3 +79,12 @@ func CleanMarkdownUsageFunc(cmd *cobra.Command) {
 		defaultHelpFunc(cmd, s)
 	})
 }
+
+// RawJSON converts an arbitrary value to json.RawMessage
+func RawJSON(value interface{}) *json.RawMessage {
+	jsonval, err := json.Marshal(value)
+	if err != nil {
+		return nil
+	}
```

shouldn't we catch an error and not silently return nil?